features/crud.feature

```diff
@@ -16,7 +16,9 @@ Feature: Create-Retrieve-Update-Delete
           "value1",
           "value2"
         ]
-      }
+      },
+      "dummyFloat": 1.2,
+      "dummyPrice": 1.2
```

Comment:
This is accepted. What should be the consistent behavior here:

- Only allowing strings when type is `Decimal`?
- Allowing javascript numbers, but also serializing `Decimal` types as numbers?